.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9].[0-9]+'  # matches to backport branches, e.g. 3.6
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
```

Comment:
It's no longer recommended to pass these: `build` is now able to build a wheel out of an sdist and not from the Git checkout meaning that it'll also test the flow that the `pip` users would hit if they disable binary installs.
```suggestion
```